Add documentation for boolean attributes (#1913)
yewstack/yew: website/docs/concepts/html/elements.md

}
```

## Boolean Attributes 

Some content attributes (e.g checked, hidden, required) are called boolean attributes. In Yew, 

> boolean attributes need to be explicitly set to 'true':

Reviewer: This should be reworded since you don't need to set them to the literal 'true', it just has to be a `bool` value.

```rust
    html! {
        <div hidden=true>
            { "This div is hidden." }
        </div>
    }
```

> This will result in the following **HTML**: 
> ```html
>     <div hidden>This div is hidden.</div>
> ```

Reviewer: This isn't exactly how Yew emits boolean attributes (blame the HTML spec) but this is much easier to understand than the real thing.
Perhaps we could clarify that the actual output would be equivalent to this so it's technically correct?

 ```suggestion
This will result in **HTML** that's functionally equivalent to this: 
```html
    <div hidden>This div is hidden.</div>
```
```

Author: Ohh I see, Yew emits attribute='attribute' from the spec :)
